packages/office-ui-fabric-react/src/components/Panel/Panel.test.tsx

```diff
@@ -25,8 +25,8 @@ describe('Panel', () => {
       dismissedCalled = true;
     };
 
-    const panel: Panel = ReactDOM.render(
-      <Panel isOpen={true} onDismiss={setDismissTrue} onDismissed={setDismissedTrue} />,
+    const panel: PanelBase = ReactDOM.render(
+      <PanelBase isOpen={true} onDismiss={setDismissTrue} onDismissed={setDismissedTrue} />,
       div
     ) as any;
 
```

Comment:
Can you add some Panel snapshots? It is not a good sign this change can be done without a single snapshot change.

Reply:
Snapshot added further up the page